### DISCUSSION TITLE
Using the muscle images

### DISCUSSION THREAD

**Discussion by travismoulton** [wger-project/wger#707] [Q&A]
Hi,
First off, huge thanks to the authors of this project. I've been making my own web app with the API and having lots of fun..
Question -  Is it possible to access the muscle images through the API? I can't seem to access them. The filepath the API returns doesn't seem to be accessible from outside. Just checking in case I'm missing something.
Thanks

**Comment by rolandgeider** [comment]
Hi there! Glad that you like it!

The paths that the API returns are relative paths, so you would need to add a `https://wger.de` to them. The images are transparent SVGs that are superimposed over a background, you can find the rest [here](https://github.com/wger-project/wger/tree/master/wger/core/static/images/muscles) (it might be easier to just download them locally since they will probably not change soon)

**Reply by travismoulton** [reply]
Thanks so much! Sorry it took me a couple days to get back to this but I appreciate the reply. I'll check it out
